usaspending_api/search/v2/views/search.py

```diff
@@ -108,8 +108,13 @@ def post(self, request):
         # 'time_period': {'fy': '2017', 'quarter': '3'},
         # 	'aggregated_amount': '200000000'
         # }]
-        for key, value in sorted(group_results.items(), key=lambda k: (ast.literal_eval(k[0])['fiscal_year'],
-                                                                       int(ast.literal_eval(k[0])[nested_order])) if nested_order else (ast.literal_eval(k[0])['fiscal_year'])):
+        flake8_nightmare = sorted(
```

Comment:
As apt as the name is, could we change it to `sorted_group_results` or the like?

Reply:
Making sure the reviewers are reviewing :)